cosmovisor/README.md

```diff
@@ -171,7 +171,7 @@ If `DAEMON_ALLOW_DOWNLOAD_BINARIES` is set to `true`, and no local binary can be
     When submitting this as a proposal ensure there are no spaces. An example command using `gaiad` could look like:
 
     ```sh
-    > gaiad tx gov submit-proposal software-upgrade Vega \
+    > gaiad tx gov submit-legacy-proposal software-upgrade Vega \
```

Comment:
this is still `gaiad tx gov submit-proposal software-upgrade Rho ` for this time, because present gaiad still use sdk0.45